src/Core/Query/LocalParameters/LocalParameters.php

```diff
@@ -744,6 +744,70 @@ public function getLocalValues(): array
         return $this->getValues(LocalParameter::TYPE_VALUE);
     }
 
+    /**
+     * @param bool $cache
+     *
+     * @throws \Solarium\Exception\OutOfBoundsException
+     *
+     * @return $this
+     */
+    public function setCache(bool $cache): self
+    {
+        return $this->addValue(LocalParameter::TYPE_CACHE, $cache ? 'true' : 'false');
```

Comment:
i think we need to clear values here first?

Otherwise calling `->setCache(true)` and then `->setCache(false)` will result in `{!cache=true,false}`? or maybe I misunderstood how it works :smile:

Reply:
yes, your're right